### PR TITLE
Bug 1246621 - Fix functional tests following e10s changes.

### PR DESCRIPTION
**Pull request by Standard8**

### CODE REVIEW

File: test/functional/serversetup.py [hunk 1]
```diff
@@ -62,8 +65,8 @@ def start_loop_server(loop_server_location):
     def start_content_server():
         content_server_location = os.environ.get('STANDALONE_SERVER')
         if content_server_location is None:
-          content_server_location = os.path.join(os.path.dirname(__file__),
-                                                 "../../standalone")
+            content_server_location = os.path.join(os.path.dirname(__file__),
```
Comment (mikedeboer) [inline test/functional/serversetup.py:68]: These are linting changes, I presume?

File: test/functional/test_1_browser_call.py [hunk 2]
```diff
@@ -81,7 +81,7 @@ def switch_to_chatbox(self):
         # for Marionette bug 1094246 to be fixed.
         chatbox = self.wait_for_element_exists(By.TAG_NAME, 'chatbox')
         script = ("return document.getAnonymousElementByAttribute("
-                  "arguments[0], 'class', 'chat-frame');")
+                  "arguments[0], 'anonid', 'content');")
```
Comment (mikedeboer) [inline test/functional/test_1_browser_call.py:84]: ...and this would be the actual fix, no?

File: test/functional/serversetup.py (hunk 1 above)
Comment (Standard8) [inline test/functional/serversetup.py:68]: Yes, just fixing the indentation.

File: test/functional/test_1_browser_call.py (hunk 2 above)
Comment (Standard8) [inline test/functional/test_1_browser_call.py:84]: Half of it, there's a similar change a bit further down. Its also why it is in a separate commit...

**Pull request merged**